src/arch/svm/SpokeUtils.ts

```diff
@@ -89,22 +90,60 @@ type ProtoFill = Omit<RelayData, "recipient" | "outputToken"> & {
   outputToken: SvmAddress;
 };
 
+export function getSlot(
+  provider: SVMProvider,
+  commitment: Commitment,
+  logger: winston.Logger,
+  maxRetries = 2
+): Promise<bigint> {
+  return _callGetSlotWithRetry(provider, commitment, 0, maxRetries, logger);
+}
+
+async function _callGetSlotWithRetry(
+  provider: SVMProvider,
+  commitment: Commitment,
+  retryAttempt: number,
+  maxRetries: number,
```

Comment:
Should we drop `maxRetries` for now, since they don't actually do anything? Otherwise it's a bit misleading.